[2023-03-12T22:42:37.448+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: validacao_pu.task_calcular_pu_externo_opq scheduled__2023-03-05T00:00:00+00:00 [queued]>
[2023-03-12T22:42:37.502+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: validacao_pu.task_calcular_pu_externo_opq scheduled__2023-03-05T00:00:00+00:00 [queued]>
[2023-03-12T22:42:37.502+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-12T22:42:37.502+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 6
[2023-03-12T22:42:37.503+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-12T22:42:37.587+0000] {taskinstance.py:1300} INFO - Executing <Task(_PythonDecoratedOperator): task_calcular_pu_externo_opq> on 2023-03-05 00:00:00+00:00
[2023-03-12T22:42:37.593+0000] {standard_task_runner.py:55} INFO - Started process 1527 to run task
[2023-03-12T22:42:37.615+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'validacao_pu', 'task_calcular_pu_externo_opq', 'scheduled__2023-03-05T00:00:00+00:00', '--job-id', '125', '--raw', '--subdir', 'DAGS_FOLDER/examples/validacao_pu.py', '--cfg-path', '/tmp/tmp3lvw_vih']
[2023-03-12T22:42:37.615+0000] {standard_task_runner.py:83} INFO - Job 125: Subtask task_calcular_pu_externo_opq
[2023-03-12T22:42:37.978+0000] {task_command.py:388} INFO - Running <TaskInstance: validacao_pu.task_calcular_pu_externo_opq scheduled__2023-03-05T00:00:00+00:00 [running]> on host 40565fc59ef6
[2023-03-12T22:42:38.452+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alexandre.yoshimatsu
AIRFLOW_CTX_DAG_ID=validacao_pu
AIRFLOW_CTX_TASK_ID=task_calcular_pu_externo_opq
AIRFLOW_CTX_EXECUTION_DATE=2023-03-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-05T00:00:00+00:00
[2023-03-12T22:42:38.458+0000] {logging_mixin.py:137} INFO - Operação: opq
[2023-03-12T22:42:38.458+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-03-12T22:42:38.496+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=validacao_pu, task_id=task_calcular_pu_externo_opq, execution_date=20230305T000000, start_date=20230312T224237, end_date=20230312T224238
[2023-03-12T22:42:38.922+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-03-12T22:42:39.040+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
